components/user-mgt/org.wso2.carbon.user.mgt/src/main/java/org/wso2/carbon/user/mgt/bulkimport/CSVUserBulkImport.java

```diff
@@ -57,6 +59,16 @@ public void addUserList(UserStoreManager userStore) throws UserAdminException {
             int duplicateCount = 0;
             while (line != null && line.length > 0) {
                 String userName = line[0];
+
+                int index;
+                index = userName.indexOf(CarbonConstants.DOMAIN_SEPARATOR);
+                if (index > 0) {
+                    String domainFreeName = userName.substring(index + 1);
+                    userName = domain + CarbonConstants.DOMAIN_SEPARATOR + domainFreeName;
```

Comment:
We can use UserCoreUtil.addDomainToName() for this purpose. It is more consistent.